Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/searchmem/mask/MnemonicSearchPlugin.java

```diff
@@ -33,7 +33,7 @@
 
 /**
  * Defines a set of actions that can be performed on a selection to initiate a memory search.  All
- * actions will ultimately open the {@link MemSearchDialog} with the search string field 
+ * actions will ultimately open the {@link ghidra.app.plugin.core.searchmem.MemSearchDialog MemSearchDialog} with the search string field 
```

Comment:
I re-looked at this one. The import is not working because MemSearchDialog isn't public. Putting in the full package didn't work either.

Reply:
You can replace the text with something like 
    <code>MemSearchDialog</code>

Reply:
I initially tried to import the package and then just use MemSearchDialog in the link but that caused the javaCompile task to fail because the class is protected.

Reply:
This was a result of how I was building it at the time. I was initially trying to limit the output so I was running the javadoc task on Base instead of running the createJavadocs task.

The documentation for this file isn't even included in the api documentation so I don't think any changes are necessary here.